.github/workflows/cuda-test.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
+on:
+  workflow_dispatch: null
+  push:
+    paths:
+      - tests/ml-test.Dockerfile
+      - .github/workflows/cuda-test.yml
```

Comment:
Please reconsider.

Reply:
walk me through your thinking?  if we change the workflow or the test, we should run the workflow, right?  or is the former implicit anyway?

Reply:
oh thanks for explaining, got it!  done now.